src/libraries/System.Globalization/tests/Invariant/InvariantMode.cs

```diff
@@ -490,13 +508,13 @@ public static IEnumerable<object[]> GetUnicode_TestData()
             yield return new object[] { "xn--de-jg4avhby1noc0d", 0, 21, "\u30D1\u30D5\u30A3\u30FC\u0064\u0065\u30EB\u30F3\u30D0" };
         }
 
-        [Fact]
+        [ConditionalFact(nameof(PredefinedCulturesOnlyIsDisabled))]
```

Comment:
Are these tests ever invoked with PredefinedCulturesOnly not disabled? It is kind of odd for every test in this assembly to be marked conditional on it - why not just not run the tests in that mode?

Reply:
Yes, it was failing for all WASM runs. Look at the discussion in the comments https://github.com/dotnet/runtime/pull/54247#issuecomment-868093885 and the reply there. If you have a better idea to handle that I can do it.

Reply:
Ok, I guess that makes sense.